vm/interpreter/src/gas_tracker/price_list.rs

```diff
@@ -322,6 +444,28 @@ impl PriceList {
     pub fn on_verify_seal(&self, _info: &SealVerifyInfo) -> GasCharge {
         GasCharge::new("OnVerifySeal", self.verify_seal_base, 0)
     }
+    #[inline]
+    pub fn on_verify_aggregate_seals(
+        &self,
+        aggregate: &AggregateSealVerifyProofAndInfos,
+    ) -> GasCharge {
+        let proof_type = aggregate.seal_proof;
+        let per_proof = self
+            .verify_aggregate_seal_per
+            .get(&proof_type)
+            .expect("There is an implementation error where proof type does not exist in table");
+        let step = self
+            .verify_aggregate_seal_steps
+            .get(&proof_type)
+            .expect("There is an implementation error where proof type does not exist in table");
```

Comment:
Same suggestion here as before. A `HashMap` is lacking the knowledge that there is a price for each variant of proof type.